docs/guides/insar_product_guide.md

```diff
@@ -1,24 +1,25 @@
 # Sentinel-1 InSAR Product Guide
+This document is a guide for users of Interferometric Synthetic Aperture Radar (InSAR) Sentinel-1 products generated by the Alaska Satellite Facility (ASF). Users can request InSAR products [On Demand](https://search.asf.alaska.edu/#/?topic=onDemand) in ASF's [Vertex](https://search.asf.alaska.edu/#/) data portal, or make use of our [Python SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) or [API](https://hyp3-docs.asf.alaska.edu/using/api/). Input pair selection in Vertex uses the [SBAS Tool](https://docs.asf.alaska.edu/vertex/sbas/). 
```

Comment:
InSAR jobs can now also be submitted from Baseline searches in addition to SBAS searches, so maybe something like:
"Input pair selection in Vertex is available using the SBAS or Basline tools"

Reply:
And might be worth spelling out "Short Baseline (SBAS) tool", but you'll know better than me how common that acronym is in the community

Reply:
Added several references to the Baseline tool.
In the expanded reference for the SBAS tool, I reference that it's for generating time series of short baseline interferograms. Do you think that's sufficient?

Reply:
Addressed more explicitly in later [commit](https://github.com/ASFHyP3/hyp3-docs/pull/96/commits/cf118cc9dca543f4fac14c15bc11c95487cd1691)